raaghu-components/src/rds-comp-icon-list/Icons.tsx

```
@@ -274,6 +276,8 @@ export const Icons: any = {
     "<svg id=\"a\" viewBox=\"0 0 21 21\"><rect id=\"b\" class=\"icon\" x=\".5\" y=\".5\" width=\"20\" height=\"20\" rx=\".919\" ry=\".919\"/><path id=\"c\" class=\"icon\" d=\"M8.134,16.075v-5.997c.015-.311,.279-.552,.59-.537,.072,.003,.144,.021,.209,.051l5.202,2.992c.272,.168,.357,.525,.189,.798-.047,.077-.112,.142-.189,.189l-5.202,3.009c-.296,.122-.635-.019-.757-.315-.024-.059-.039-.122-.043-.186v-.003Z\"/><line id=\"d\" class=\"icon\" x1=\".822\" y1=\"5.8\" x2=\"20.177\" y2=\"5.8\"/><line id=\"e\" class=\"icon\" x1=\"5.8\" y1=\"5.799\" x2=\"7.919\" y2=\"1.053\"/><line id=\"f\" class=\"icon\" x1=\"12.85\" y1=\"5.799\" x2=\"15.016\" y2=\"1.053\"/></svg>",
   plus:
     "<svg id=\"a\" viewBox=\"0 0 21 21\"><defs></defs><line id=\"b\" class=\"icon\" x1=\".5\" y1=\"10.5\" x2=\"20.5\" y2=\"10.5\"/><line id=\"c\" class=\"icon\" x1=\"10.5\" y1=\".5\" x2=\"10.5\" y2=\"20.5\"/></svg>",
+  plus_color:
```

ESLint

[error] Expected indentation of 4 spaces but found 2.

(indent)